Frameworks/include/WRLHelpers.h

```diff
@@ -156,7 +156,7 @@ HRESULT ForEach(ABI::Windows::Foundation::Collections::IMap<K, V>* map, Q&& func
 }
 
 #pragma region Async Helpers
-struct ActionCallback : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::ClassicCom>,
+struct ActionCallback : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::WinRtClassicComMix>,
```

Comment:
No worries.  Thanks!

---
In reply to: [87461400](https://github.com/Microsoft/WinObjC/pull/1344#discussion_r87461400) [](ancestors = 87461400)